functions.php

```diff
@@ -225,7 +220,12 @@ function twentyseventeen_scripts() {
 
 	wp_enqueue_script( 'twentyseventeen-navigation', get_template_directory_uri() . '/assets/js/navigation.js', array(), '20151215', true );
 
-	wp_enqueue_script( 'twentyseventeen-global', get_template_directory_uri() . '/assets/js/global.js', array( 'jquery' ), '20151215', true );
+	wp_localize_script( 'twentyseventeen-navigation', 'screenReaderText', array(
+		'expand'   => __( 'expand child menu', 'twentyseventeen' ),
+		'collapse' => __( 'collapse child menu', 'twentyseventeen' ),
```

Comment:
Sorry for the delay! I needed to investigate this one a bit - the same lowercase string appears in Twenty Fifteen and Twenty Sixteen, so I wanted to make sure it was okay to replace.

The string was originally set to be lowercase based on this issue: https://core.trac.wordpress.org/ticket/30083#comment:7; in comment 9, it was noted the all-caps actually came from the CSS, not the string's formatting, but the string was left as lowercase just in case.

I double-checked with @davidakennedy and it looks like it'll be ok to capitalize the first word in these strings. That will be more consistent with other strings in the theme, and shouldn't cause issues with how the screen reader reads it.